gitlab/v4/objects.py

```diff
@@ -2035,6 +2110,7 @@ class Project(SaveMixin, ObjectDeleteMixin, RESTObject):
         ('notificationsettings', 'ProjectNotificationSettingsManager'),
         ('pipelines', 'ProjectPipelineManager'),
         ('protectedbranches', 'ProjectProtectedBranchManager'),
+        ('pipeline_schedules', 'ProjectPipelineSchedulesManager'),
```

Comment:
Could you use `pipelineschedules` for consistency with the rest of the attribute names?